Update toZonedDateTime documentation. (#1149)
tc39/proposal-temporal: docs/instant.md

timestamp.toZonedDateTimeISO('-08:00'); // => 2019-03-30T16:45-08:00[-08:00]
```

### instant.**toZonedDateTime**(_item_: object) : Temporal.ZonedDateTime

**Parameters:**

- `item` (object): an object with properties to be added to `time`. The following properties are
  recognized:

>   - `calendar` (required `Temporal.Calendar` or value convertible to one, or an object implementing the calendar protocol):

Reviewer: ```suggestion
  - `calendar` (required calendar identifier string, `Temporal.Calendar` object, or object implementing the calendar protocol):
```

I found "value convertible to one" to be confusing because 99.9% of the time that value will be an ID string. I'd suggest we just say "string" here to be clearer.

    the calendar in which to interpret `instant`.

>   - `timeZone` (required `Temporal.TimeZone` or value convertible to one, or an object implementing the [time zone protocol](./timezone.md#protocol)):

Reviewer: ```suggestion
  - `timeZone` (required time zone identifier string, `Temporal.TimeZone` object, or object implementing the [time zone protocol](./timezone.md#protocol)):
```

Same.
